tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -10,9 +10,35 @@ use tfhe_cuda_backend::cuda_bind::{
     cuda_synchronize_device,
 };
 
+use tfhe_cuda_backend::cuda_bind::cuda_get_number_of_gpus;
+
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct GpuIndex(pub u32);
 
+impl GpuIndex {
+    pub fn num_gpus() -> u32 {
```

Comment:
Why have this function as a method of `GpuIndex`?

Reply:
Did not really know where to put this

Reply:
There is already `get_numbers_of_gpus` available tfhe/src/core_crypto/gpu/mod.rs:729.

Reply:
I don't see it

Reply:
Oh well I do now, but grep did not 🤔